docs/docsite/.templates/eol_banner.html

```diff
@@ -0,0 +1,6 @@
+{# Creates a banner at the top of the page for EOL versions. #}
+<div id="banner" class="Admonition caution">
+    <p>
+      You are reading an unmaintained version of the Ansible documentation. Unmaintained Ansible versions can contain unfixed security vulnerabilities (CVE). Please upgrade to a maintained version. See <a href="/ansible/latest/">the latest Ansible documentation</a>.
+    </p>
```

Comment:
Bah, I forgot that I wanted to update this template to list the active versions. It should be possible to do that now instead of just saying "Please upgrade to a maintained version." Versions should be available in the template via the extension.

I'll make that change later though.

Reply:
In the end I decided against this. It was probably overcomplicating things and might mean we'd need to update EOL docs to reflect new active versions.